primer/brand
# Add Tooltip component

Closes https://github.com/github/primer/issues/3060

## Summary

Adds a Tooltip component for showing contextual information when focusing or hovering interactive elements. I made the decision to remove the caret shown in the screenshot in the issue's description. Other Primer designers were happy with this, and it's the same style we use in `@primer/react`'s new Tooltip.

## List of notable changes:

**added** Tooltip component



-
-

## What should reviewers focus on?

Confirm that the Tooltip looks and behaves as expected.



- 
-

## Steps to test:

1. Open the Tooltip component in the CI-deployed preview environment
2. Check out all of the features demo'd in the "Features" directory or use Storybook's controls to play with the "Playground" story
3. Verify that the Tooltip behaves as described in the following issue.





## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

N/A

 </td>
<td valign="top">

https://github.com/primer/brand/assets/2313998/bfe95579-059e-48bb-b1a3-a15c95f55f43

</td>
</tr>
</table>
